File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/openTripMap/Places.java
package com.tammamkhalaf.myuaeguide.categories.openTripMap;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Places {

    @SerializedName("kinds")
    @Expose
    private String kinds;
    @SerializedName("sources")
    @Expose
    private Sources sources;
    @SerializedName("bbox")
    @Expose
    private Bbox bbox;
    @SerializedName("point")
    @Expose
    private Point point;
    @SerializedName("osm")
    @Expose
    private String osm;
    @SerializedName("otm")
    @Expose
    private String otm;
    @SerializedName("xid")
    @Expose
    private String xid;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("wikipedia")
    @Expose
    private String wikipedia;
    @SerializedName("image")
    @Expose
    private String image;
    @SerializedName("wikidata")
    @Expose
    private String wikidata;
    @SerializedName("rate")
    @Expose
    private String rate;
    @SerializedName("info")
    @Expose
    private Info info;

    public String getKinds() {
        return kinds;
    }

    public void setKinds(String kinds) {
        this.kinds = kinds;
    }

    public Sources getSources() {
        return sources;
    }

    public void setSources(Sources sources) {
        this.sources = sources;
    }

    public Bbox getBbox() {
        return bbox;
    }

    public void setBbox(Bbox bbox) {
        this.bbox = bbox;
    }

    public Point getPoint() {
        return point;
    }

    public void setPoint(Point point) {
        this.point = point;
    }

    public String getOsm() {
        return osm;
    }

    public void setOsm(String osm) {
        this.osm = osm;
    }

    public String getOtm() {
        return otm;
    }

    public void setOtm(String otm) {
        this.otm = otm;
    }

    public String getXid() {
        return xid;
    }

    public void setXid(String xid) {
        this.xid = xid;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getWikipedia() {
        return wikipedia;
    }

    public void setWikipedia(String wikipedia) {
        this.wikipedia = wikipedia;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getWikidata() {
        return wikidata;
    }

    public void setWikidata(String wikidata) {
        this.wikidata = wikidata;
    }

    public String getRate() {
        return rate;
    }

    public void setRate(String rate) {
        this.rate = rate;
    }

    public Info getInfo() {
        return info;
    }

    public void setInfo(Info info) {
        this.info = info;
    }

}

File: shared/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/openTripMap/Places.kt
package com.tammamkhalaf.myuaeguide.categories.openTripMap

import com.google.gson.annotations.Expose

class Places {
    @SerializedName("kinds")
    @Expose
    private var kinds: String? = null

    @SerializedName("sources")
    @Expose
    private var sources: com.tammamkhalaf.myuaeguide.categories.openTripMap.Sources? = null

    @SerializedName("bbox")
    @Expose
    private var bbox: com.tammamkhalaf.myuaeguide.categories.openTripMap.Bbox? = null

    @SerializedName("point")
    @Expose
    private var point: com.tammamkhalaf.myuaeguide.categories.openTripMap.Point? = null

    @SerializedName("osm")
    @Expose
    private var osm: String? = null

    @SerializedName("otm")
    @Expose
    private var otm: String? = null

    @SerializedName("xid")
    @Expose
    private var xid: String? = null

    @SerializedName("name")
    @Expose
    private var name: String? = null

    @SerializedName("wikipedia")
    @Expose
    private var wikipedia: String? = null

    @SerializedName("image")
    @Expose
    private var image: String? = null

    @SerializedName("wikidata")
    @Expose
    private var wikidata: String? = null

    @SerializedName("rate")
    @Expose
    private var rate: String? = null

    @SerializedName("info")
    @Expose
    private var info: com.tammamkhalaf.myuaeguide.categories.openTripMap.Info? = null
    fun getKinds(): String? {
        return kinds
    }

    fun setKinds(kinds: String?) {
        this.kinds = kinds
    }

    fun getSources(): com.tammamkhalaf.myuaeguide.categories.openTripMap.Sources? {
        return sources
    }

    fun setSources(sources: com.tammamkhalaf.myuaeguide.categories.openTripMap.Sources?) {
        this.sources = sources
    }

    fun getBbox(): com.tammamkhalaf.myuaeguide.categories.openTripMap.Bbox? {
        return bbox
    }

    fun setBbox(bbox: com.tammamkhalaf.myuaeguide.categories.openTripMap.Bbox?) {
        this.bbox = bbox
    }

    fun getPoint(): com.tammamkhalaf.myuaeguide.categories.openTripMap.Point? {
        return point
    }

    fun setPoint(point: com.tammamkhalaf.myuaeguide.categories.openTripMap.Point?) {
        this.point = point
    }

    fun getOsm(): String? {
        return osm
    }

    fun setOsm(osm: String?) {
        this.osm = osm
    }

    fun getOtm(): String? {
        return otm
    }

    fun setOtm(otm: String?) {
        this.otm = otm
    }

    fun getXid(): String? {
        return xid
    }

    fun setXid(xid: String?) {
        this.xid = xid
    }

    fun getName(): String? {
        return name
    }

    fun setName(name: String?) {
        this.name = name
    }

    fun getWikipedia(): String? {
        return wikipedia
    }

    fun setWikipedia(wikipedia: String?) {
        this.wikipedia = wikipedia
    }

    fun getImage(): String? {
        return image
    }

    fun setImage(image: String?) {
        this.image = image
    }

    fun getWikidata(): String? {
        return wikidata
    }

    fun setWikidata(wikidata: String?) {
        this.wikidata = wikidata
    }

    fun getRate(): String? {
        return rate
    }

    fun setRate(rate: String?) {
        this.rate = rate
    }

    fun getInfo(): com.tammamkhalaf.myuaeguide.categories.openTripMap.Info? {
        return info
    }

    fun setInfo(info: com.tammamkhalaf.myuaeguide.categories.openTripMap.Info?) {
        this.info = info
    }
}